src/containers/FrontpageEditPage/FrontpageArticleSearch.tsx

```diff
@@ -40,6 +40,19 @@ const PopoverContent = styled(Content)`
 const FrontpageArticleSearch = ({ articleId, children, onChange }: Props) => {
   const { t } = useTranslation();
   const { values } = useFormikContext<MenuWithArticle>();
+  const [windowHeight, setWindowHeight] = useState<number>(window.innerHeight);
+
+  useEffect(() => {
```

Comment:
Denne kjører alltid, ikkje sant? Kunne du hatt [window.innerHeight] som sjekk i useEffect? Eller har det ikkje noko for seg?

Reply:
@gunnarvelle Jeg tenkte også det, men når jeg satte en console.log i den for å teste, printa den uansett kun ved load og når jeg aktivt resizet. Jeg gjorde for ordens skyld også en lokal test hvor jeg printet med og uten dependency array, og resultatet ble ganske likt (sakte decrease og sakte increase av window height til ca samme plass for begge, ga ca ~350 logs for begge). Når jeg ikke aktivt resizet, printet ingenting.
Jeg prøvde å legge den til i dependency array allikevel (fordi jeg tenkte det var et poeng at den var sporbar), men det ville ikke ESLint ha noe av:
![image](https://github.com/NDLANO/editorial-frontend/assets/33912044/88157d97-befe-4a3f-bbb5-4d4fbc9b4a82)

Og sida det ikke hadde noen praktisk betydning, så langt jeg kunne se, har jeg derfor tatt den ut igjen.